src/Symfony/Component/Workflow/Validator/WorkflowValidator.php

```diff
@@ -20,5 +20,6 @@ class WorkflowValidator implements DefinitionValidatorInterface
 {
     public function validate(Definition $definition, $name)
     {
+        return true;
```

Comment:
I think I tend to agree with @ro0NL that this method shouldn't return anything. Why is that method empty in the `WorkfowValidator` at all?